package.json

```diff
@@ -34,8 +34,8 @@
   "scripts": {
     "gen:stacksContract": "rm -rf generated/smartContract && mkdir -p generated/smartContract && tsx ./scripts/generateClarityTranscoders.ts",
     "gen": "pnpm run gen:stacksContract",
-    "docs": "typedoc",
-    "docs:watch": "typedoc --watch",
+    "docs": "typedoc src/index.ts",
```

Comment:
we have already added `typedoc.config.js`, no need to pass the specific file path to typedoc

Reply:
Yes, but the way we have implemented it, much more documentation is generated in Typedoc. Here is a link for you to compare how it looks: https://tenuki.github.io/xlink-sdk/

Reply:
do you mean the `/bitcoinHelpers` export? what's what I want, the content of `/bitcoinHelpers` really are functions of what we offer

Reply:
Somehow, without this change (adding src/index.ts) it doesn't renders all the items on this left column but only just the first 3 ![image](https://github.com/user-attachments/assets/1f980091-e7ac-4bbb-8db6-51af23c9a655)

To render them all it is required the src/index.ts

Reply:
You are right; the current version of the typedoc.config.js file may not be so appropriate; we might need to change it to:

```typescript
const entryPoints = Object.keys(pkgInfo.exports).map(i => {
  const exportEndpoint = i === "." ? "./index" : I
  const exportEndpointFilePath = path.join("../src", exportEndpoint) + ".ts"
  return exportEndpointFilePath
})
```

Reply:
Thanks we are addressing everything of this!!